# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/reftests/filters/backdrop-filter-culled-filter-ref.yaml
# Commit: 51ba9e64ff0a
# Full Hash: 51ba9e64ff0a4071cdd57dbf24110606faad781e
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-05-27 15:58:57
# Description:
#   Bug 1771388 - Ensure backdrop capture has same clip as backdrop render r=gfx-reviewers,lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D147482
# ==============================================================================

diff -r 5e2d37a71968 -r 51ba9e64ff0a gfx/wr/wrench/reftests/filters/backdrop-filter-culled-filter-ref.yaml
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/gfx/wr/wrench/reftests/filters/backdrop-filter-culled-filter-ref.yaml	Fri May 27 05:38:29 2022 +0000
@@ -0,0 +1,5 @@
+---
+root:
+  items:
+    - image: checkerboard(2,14,14,14,14)
+      bounds: 0 0 200 200